pl_examples/domain_templates/imagenet.py

```diff
@@ -70,9 +65,8 @@ def __init__(
         lr: float = 0.1,
         momentum: float = 0.9,
         weight_decay: float = 1e-4,
-        batch_size: int = 4,
-        workers: int = 2,
-        **kwargs,
+        batch_size: int = 256,
+        workers: int = 4,
```

Comment:
These now match the values in https://github.com/pytorch/examples/blob/master/imagenet/main.py